lib/parzival_web/live/tools/announcement_live/index.ex

```diff
@@ -6,33 +6,15 @@ defmodule ParzivalWeb.AnnouncementLive.Index do
 
   @impl true
   def mount(_params, _session, socket) do
-    {:ok, assign(socket, :announcements, list_announcements())}
+    {:ok, socket}
   end
 
   @impl true
   def handle_params(params, _url, socket) do
```

Comment:
The variable "params" is unused, you should use an underscore.

Reply:
:pray: